ChangeLog.d/ecb_iv_fix.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Correct the default iv size for mbed_cipher_info_t structures using
+     MBEDTLS_MODE_ECB to 0, since ecb mode ciphers don't use ivs.
```

Comment:
Nitpick: ECB instead of `ecb` and while at it probably that "IV" is better than "iv".

Reply:
Done.